grizli/data/auto_script_defaults.yml

```diff
@@ -10,7 +10,7 @@ filters: &filters
     [F410M, F467M, F547M, F550M, F621M, F689M, F763M, F845M, F200LP, F350LP,
      F435W, F438W, F439W, F450W, F475W, F475X, F555W, F569W, F600LP, F606W,
      F622W, F625W, F675W, F702W, F775W, F791W, F814W, F850LP, G800L, F098M,
-     F127M, F139M, F153M, F105W, F110W, F125W, F140W, F160W, G102, G141]
+     F127M, F139M, F153M, F105W, F110W, F125W, F140W, F160W, G102, G141, GR150C, F115W]
```

Comment:
Add additional NIRISS and NIRCAm grisms and filters?

Reply:
Added more filter names for NIRISS/NIRCam here: 4323a2902fda32fc117da85d0494d7b20fcd33e8